### ISSUE TITLE
No indication that messages of a task are loaded on task details page

### ISSUE THREAD

**Issue by mbukatov** [Tendrl/ui#620]
## Description

There is no indication that messages of a task are loaded on task details page.

## Version

Latest snapshot build from master branch (part of [upcoming 1.5.2 version](https://github.com/Tendrl/specifications/milestone/1)):

```
tendrl-ui-1.5.2-20170916T043442.5764617.noarch
```

Other related packages:

```
tendrl-commons-1.5.2-20170916T041545.0eb5c3f.noarch
tendrl-api-1.5.2-20170916T041838.48452aa.noarch
tendrl-node-agent-1.5.2-20170916T041433.436cac0.noarch
tendrl-monitoring-integration-1.5.2-20170918T072405.e742bba.noarch
tendrl-grafana-plugins-1.5.2-20170918T072405.e742bba.noarch
tendrl-api-httpd-1.5.2-20170916T041838.48452aa.noarch
etcd-3.1.9-2.el7.x86_64
grafana-4.5.1-1.x86_64
graphite-web-0.9.16-1.el7.noarch
```

## Steps to Reproduce

1. Prepare machines with GlusterFS cluster, including gluster volume (I used [nightly builds](https://github.com/usmqe/usmqe-setup/tree/master/roles/gluster-centos-repo) and [volume_usmqe_alpha_distrep_4x2.create.conf](https://github.com/usmqe/usmqe-setup/blob/master/gdeploy_config/volume_usmqe_alpha_distrep_4x2.create.conf))
2. Install Tendrl via tendrl-ansible there, using [snapshot builds](https://copr.fedorainfracloud.org/coprs/tendrl/tendrl/)
3. Log into the Tendrl web interface as an admin user
4. Import cluster via Tendrl web ui and wait until the task finishes with success
5. Click on Tasks item and then on the just finished import cluster task
6. See how the task details page (which you asked for in the previous step) is loaded

## Actual Results

1. At first, you see a spinner icon indicating that the data are loading.
2. Then the header with Task ID, timestamp and status is shown, but there is no indication that messages are being loaded (see that there is a pending `/messages` api call)
3. After some additional time, messages are finally displayed

![screenshot_20170918_174926](https://user-images.githubusercontent.com/4759819/30551144-d4d65490-9c99-11e7-8e1d-4278e148d4a3.png)

## Expected Results

1. At first, you see a spinner icon indicating that the data are loading.
2. Then the header with Task ID, timestamp and status is shown and there is another spinner icon for the messages, which are being loaded
3. After some additional time, messages are finally displayed

## Additional Details

This could be extra confusing considering that the messages will be deleted after some time, see https://github.com/Tendrl/api/issues/219

**Comment by mbukatov** [comment]
I have a screen recording of the issue, ping me for access if needed.

**Comment by a2batic** [comment]
@mbukatov Its fixed in tendrl-ui-1.5.2-20170920T073234.f7b6ad2.noarch.rpm. Please verify.

**Comment by mbukatov** [comment]
@a2batic I enabled etcd auth and retried with `tendrl-ui-1.5.2-20170920T143459.2bcbd93.noarch` and the spinner is shown as expected.
